include/aspect/particle/interpolator/bilinear_least_squares.h

```diff
@@ -54,6 +54,25 @@ namespace aspect
 
           // avoid -Woverloaded-virtual:
           using Interface<dim>::properties_at_points;
+
+          /**
+           * Declare the parameters this class takes through input files.
+           */
+          static
+          void
+          declare_parameters (ParameterHandler &prm);
+
+          /**
+           * Read the parameters this class declares from the parameter file.
+           */
+          virtual
+          void
+          parse_parameters (ParameterHandler &prm);
+
+        private:
+          bool use_global_valued_limter;
```

Comment:
Sorry, I should have been more explicit: There is a typo in the variable name `limter` -> `limiter` ... I did not mean to question the name itself :smile: